sklearn/naive_bayes.py

```diff
@@ -741,6 +741,11 @@ class MultinomialNB(_BaseDiscreteNB):
     C.D. Manning, P. Raghavan and H. Schuetze (2008). Introduction to
     Information Retrieval. Cambridge University Press, pp. 234-265.
     https://nlp.stanford.edu/IR-book/html/htmledition/naive-bayes-text-classification-1.html
+
+    .. versionchanged:: v0.16
```

Comment:
This can't be in References. and I'm not sure whether we want this level of detail, if it implies maintaining such per-estimator changelogs elsewhere (which would be neat but is a lot of work)